cloud_functions/token_refresh/main.py

```
@@ -0,0 +1,108 @@
+
+    # Add CORS headers to the response
+    headers = {
+        'Access-Control-Allow-Origin': origin,
+        'Access-Control-Allow-Methods': 'POST, OPTIONS',
+        'Access-Control-Allow-Headers': 'Authorization, Content-Type',
+    }
+
+    return headers
+
+@functions_framework.http
+def refresh_token(request):
```

Ruff (0.8.2)

`refresh_token` is too complex (13 > 10)

(C901)